src/main/java/network/brightspots/rcv/Utils.java

```diff
@@ -86,4 +91,41 @@ static String getUserName() {
   static String[] splitByNewline(String s) {
     return s.trim().split("\\s*\\r?\\n\\s*");
   }
+
+  static byte[] readFileToByteArray(File file) {
+    byte[] fileBytes = null;
+    try {
+      fileBytes = Files.readAllBytes(file.toPath());
```

Comment:
Probably not necessary for now, but ideally we would have a way to compute this without reading the entire file into memory at once. Maybe something like what's discussed here would work? https://stackoverflow.com/questions/76013476/java-inputstream-copy-to-and-calculate-hash-at-the-same-time

Reply:
Good call -- I imagine at some point we may decide to hash the input files too, so I've updated this function to read the file in 1kb chunks. Can you take another quick look before I merge?